get-compose-running.sh

```diff
@@ -21,6 +21,15 @@ containers_running() {
         healthy=$(docker ps -q --filter "id=${id}" --filter "health=healthy" | wc -l)
         is_ok=$((noHealth + healthy))
         num_ok=$((num_ok + is_ok))
+        is_dead=$((1 - noHealth + healthy))
+
+        if [ ${is_dead} -eq 1 ]; then
+            consumer_name=$(docker ps -a --filter="id=1992a8399583" --format="{{ .Names }}")
```

Comment:
What does this id mean?

Reply:
This means that I didn't replace the constant to a variable.